docs/NMOS With JPEG XS.md

```diff
@@ -123,9 +126,10 @@ An example Receiver resource is provided in the [Examples](../examples/) that de
 ## JPEG XS IS-05 Senders and Receivers
 
 Connection Management using IS-05 proceeds in exactly the same manner as for any other stream format carried within RTP.
-The SDP file at the **/transportfile** endpoint on Senders MUST comply with the requirements of RFC 9134 and, if appropriate, ST 2110-22.
+The SDP file at the **/transportfile** endpoint on Senders MUST comply with the same requirements described for the SDP file at the IS-04 Sender `manifest_href`.
 
 An SDP file provided in the `transport_file` attribute of a `PATCH` request on the **/staged** endpoint of Receivers MUST also comply with RFC 9134 and, if appropriate, ST 2110-22.
+Note that Receivers MUST handle such SDP files that do not comply with the additional requirements described for Senders.
```

Comment:
```suggestion
However, a Receiver MUST handle such an SDP file that does not comply with the additional requirements described for Senders.
When processing a request, a Receiver can/MAY/SHOULD use the parameters in the SDP file to evaluate if it would not be capable of consuming the stream.
```